src/sbvr-api/sbvr-utils.ts

```diff
@@ -676,18 +676,19 @@ export const executeModels = async (
 				api[apiRoot].logger = { ...console };
```

Comment:
Avoiding this awkward monkey patching of the pinejs client instance is the main benefit, since it won't persist on a `.clone` or similar and generally makes typing life there a lot harder because it means we get the following issues:
```ts
const clone = loggingClient.clone()
clone.logger.log() // Cannot call `log` of undefined
// typeof clone !== typeof loggingClient
```
and makes typing around cloning just an unnecessary pain